nerfstudio/models/nerfacto.py

```diff
@@ -95,6 +106,10 @@ class NerfactoModelConfig(ModelConfig):
     """Max num iterations for the annealing function."""
     use_single_jitter: bool = True
     """Whether use single jitter or not for the proposal networks."""
+    use_pred_normals: bool = False
+    """Whether to predict normals or not."""
+    compute_normals: bool = False
+    """Whether to compute normals from density gradient or not."""
```

Comment:
Can delete this, and only compute the analytic normals when `use_pred_normals = True` because it's needed for supervision.